lib/active_model_serializers/model.rb

```diff
@@ -3,42 +3,50 @@
 # serializable non-activerecord objects.
 module ActiveModelSerializers
   class Model
-    include ActiveModel::Model
     include ActiveModel::Serializers::JSON
+    include ActiveModel::Validations
+    include ActiveModel::Conversion
+    extend ActiveModel::Naming
+    extend ActiveModel::Translation
```

Comment:
code adapted from ActiveModel::Model which is has good logic in it, but doesn't still has a few cases we want to handle differently